.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+## Contributor Agreement
+
+- [ ] I confirm I have signed the [Contributor License Agreement](https://github.com/overleaf/overleaf/blob/master/CONTRIBUTING.md#contributor-license-agreement)
```

Comment:
There is nothing to sign, probably "read" instead?

```suggestion
- [ ] I confirm I have read the [Contributor License Agreement](https://github.com/overleaf/overleaf/blob/master/CONTRIBUTING.md#contributor-license-agreement)
```

Reply:
If you follow it there's a link to a form where you can sign the agreement.

Reply:
Oh saw it now, thanks!